phpBB/styles/prosilver/template/ucp_pm_viewfolder.html

```diff
@@ -1,130 +1,35 @@
-<!-- INCLUDE ucp_header.html -->
-
-<!-- IF not PROMPT -->
-	<!-- INCLUDE ucp_pm_message_header.html -->
-<!-- ENDIF -->
-
-<!-- IF PROMPT -->
-	<h2>{L_EXPORT_AS_CSV}</h2>
-	<form id="viewfolder" method="post" action="{S_PM_ACTION}">
-	<div class="panel">
-		<div class="inner">
-		<h3>{L_OPTIONS}</h3>
-		<fieldset>
-			<dl>
-				<dt><label for="delimiter">{L_DELIMITER}{L_COLON}</label></dt>
-				<dd><input class="inputbox" type="text" id="delimiter" name="delimiter" value="," /></dd>
-			</dl>
-			<dl>
-				<dt><label for="enclosure">{L_ENCLOSURE}{L_COLON}</label></dt>
-				<dd><input class="inputbox" type="text" id="enclosure" name="enclosure" value="&#034;" /></dd>
-			</dl>
-		</fieldset>
-		</div>
-	</div>
-	<fieldset class="submit-buttons">
-		<input type="hidden" name="export_option" value="CSV" />
-		<input class="button1 button button-form" type="submit" name="submit_export" value="{L_EXPORT_FOLDER}" />&nbsp;
-		<input class="button1 button button-form-bold" type="reset" value="{L_RESET}" name="reset" />&nbsp;
-		{S_FORM_TOKEN}
-	</fieldset>
-	</form>
-
-<!-- ELSE -->
-
-	<!-- IF NUM_REMOVED -->
-		<div class="notice">
-			<p>{RULE_REMOVED_MESSAGES}</p>
-		</div>
-	<!-- ENDIF -->
-
-	<!-- IF NUM_NOT_MOVED -->
-		<div class="notice">
-			<p>{NOT_MOVED_MESSAGES}<br />{RELEASE_MESSAGE_INFO}</p>
-		</div>
-	<!-- ENDIF -->
-
-	<!-- IF .messagerow -->
-		<ul class="topiclist two-columns">
-			<li class="header">
-				<dl>
-					<dt><div class="list-inner">{L_MESSAGE}</div></dt>
-					<dd class="mark">{L_MARK}</dd>
-				</dl>
+{% include ('ucp_header.html') %}
+
+{% if not PROMPT %}
+	{% include('ucp_pm_message_header.html') %}
+{% endif %}
+
+
+{% if messagerow %}
+	<div class="main">
+		<div class="left-panel">
+
+		{% for messagerow in messagerow %}
+			<li class="message-row{% if messagerow.S_ROW_COUNT is odd %} row1{% else %} row2{% endif %}{% if messagerow.FOLDER_IMG_STYLE == 'pm_unread' %} msg-unread{% endif %}">
```

Comment:
Not sure if it is possible to make whole `<li>` clickable (except for recipients of course). @hanakin is it valid to wrap `<div>` by `<a>`? Or we need to use JS on `<div>` to perform such effect?